guide/contributing/developer-guide.md

```diff
@@ -63,6 +64,12 @@ Automated component versioning is facilitated by the [Conventional Commits speci
 be deliberate when choosing the type of commit. For example, only use the "feat" type if you are adding a backwards-compatible feature
 to a component.
 
+Due to the subjectivity of versioning front-end components, keep these suggestions in mind when making conventional commits:
+
+* **Breaking changes** are removals of features such as props, changes that affect the [box model](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model), or dramatic changes in branding or appearance
+* **Minor changes** are new features, animations, props, or visual options
+* **Patches** are defect fixes that do not remove features, alter pixel dimensions related to the box model, nor add new features. If an intended feature was not working in a previous release, changing that feature to match the original design counts as a patch even if it affects the box model
+
```

Comment:
Is the intention here to explain the commit process or the publishing conventions? If it is committing, perhaps you should use the `fix` and `feat` language here. When I'm making a commit I don't use the "minor change" or "patch"language. Major, minor, and patch relate to versioning, which may not be appropriate here in the committing docs.